### DISCUSSION TITLE
Welcome to orgweb07 Discussions!

### DISCUSSION THREAD

**Discussion by webdev0715** [orgweb07/orgweb07#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mdheisser** [comment]
Perfect

**Reply by webdev0715** [reply]
perfect

**Reply by mdheisser** [reply]
Nice

**Comment by mdheisser** [comment, marked as answer]
Good
